membersince/membersince.php

```diff
@@ -16,13 +17,11 @@ function membersince_install()
 	Addon::registerHook('profile_advanced', 'addon/membersince/membersince.php', 'membersince_display');
 }
 
-function membersince_uninstall()
-{
+function membersince_uninstall() {
```

Comment:
Please leave the new line before the function opening brace.

Reply:
I also fixed the one above.